pkg/scheduler/testing/wrappers.go

```diff
@@ -170,6 +172,19 @@ func (p *PodWrapper) Namespace(s string) *PodWrapper {
 	return p
 }
 
+// OwnerReference updates the owning controller of the pod.
+func (p *PodWrapper) OwnerReference(name string, gvk schema.GroupVersionKind) *PodWrapper {
+	p.OwnerReferences = []metav1.OwnerReference{
```

Comment:
Given the name is `OwnerReference`, I'd tend to append the given gvk to the OwnerReferences slice; instead of set it - just in case someday we need to build a case to set multiple references.

BTW: shorten the name to `OwnerRef` to save a few characters.

Reply:
can I chose not to do any? I feel this interface is good enough :)

Reply:
Sure, we can update once we have a real requirement.